shared/dev/dumb-sheet.render.desktop.js

```diff
@@ -52,6 +53,10 @@ class Render extends Component<void, Props, any> {
           const items = Object.keys(map.mocks)
             .filter(mockKey => !filter || includeAllChildren || (key.toLowerCase() + mockKey.toLowerCase()).indexOf(filter) !== -1)
             .map((mockKey, idx) => {
+              --numItemsLeftWeCanShow
```

Comment:
well the count applies across the nested map so the math is a little trickier. i did this since its a lot simpler